packages/wasm/package.json

```diff
@@ -5,6 +5,8 @@
   "license": "MIT",
   "type": "module",
   "scripts": {
+    "download-keys": "tsx ../../apps/extension/src/utils/download-proving-keys.ts",
```

Comment:
shouldn't be referring to a script in another package - this package should contain the script

Reply:
moved script to `wasm` package